misc/openapi-ballerina/modules/openapi-validator/src/main/java/org/ballerinalang/openapi/validator/OpenAPIValidatorPlugin.java

```diff
@@ -136,54 +139,30 @@ public void process(ServiceNode serviceNode, List<AnnotationAttachmentNode> anno
                                     }
                                 } else {
                                     dLog.logDiagnostic(Diagnostic.Kind.ERROR, annotation.getPosition(),
-                                                       "Contract path should be applied as a string value");
+                                            "Contract path should be applied as a string value");
```

Comment:
these can be moved into the error messages class I think... it will be much clear then....